src/config.rs

```diff
@@ -0,0 +1,135 @@
+use anyhow::{anyhow, Result};
+use clap::{App, Arg};
+
+use crate::runtime;
+
+pub const VERSION: &str = "0.4.0";
+
+#[derive(Debug)]
+pub struct Config {
```

Comment:
I think `config::Config` isn't an ideal import path, but nothing better comes to mind at the moment. We could refactor in a follow-up.

Reply:
I couldn't think of anything better either. If it annoys us a bunch in the future we can switch it up.